setup.py

```diff
@@ -153,7 +153,12 @@ def write_version_py(filename=_VERSION_FILENAME):
         install_requires = __requires__,
         license = "BSD",
         packages = find_packages(),
-        package_data = {'': ['images/*', '*.ini',]},
+        package_data = {'': ['images/*', '*.ini',],
```

Comment:
Specifying it only in MANIFEST.in as `include tox.ini` may be a better approach.